set.mm

```diff
@@ -409565,8 +409616,9 @@ is a bijection ( 1-1 onto function) in a simple graph.  (Contributed
       KULUM $.
 
     $d E v $.
-    $( If a graph/class has no edges, it has universal vertices if and only if
-       it has exactly one vertex.  (Contributed by AV, 30-Oct-2020.) $)
+    $( TODO-AV: antecedent ` G e. W ` can be removed!  If a graph/class has no
```

Comment:
Ok, but try not to add too many TODOs and exclamation points in Main, or not for too long (the TODOs are ok in mathboxes).

Reply:
This theorem was already in main (and also used in proofs of theorems in main), therefore I had to tag it here. There are other TODOs in main, some still from Norm. Of course, such TODOs should be dealt with soon (as I will do with this one).